<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewTorrentDialog</class>
 <widget class="QDialog" name="AddNewTorrentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>620</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="AddNewTorrentDialogLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QFrame" name="torrentoptionsFrame">
      <layout class="QVBoxLayout" name="mainlayout_addui">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="managementLayout">
         <item>
          <widget class="QLabel" name="labelTorrentManagementMode">
           <property name="text">
            <string>Torrent Management Mode:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboTTM">
           <property name="toolTip">
            <string>Automatic mode means that various torrent properties(eg save path) will be decided by the associated category</string>
           </property>
           <item>
            <property name="text">
             <string>Manual</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Automatic</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxSavePath">
         <property name="title">
          <string>Save at</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="FileSystemPathComboEdit" name="savePath" native="true"/>
          </item>
          <item alignment="Qt::AlignRight">
           <widget class="QCheckBox" name="checkBoxRememberLastSavePath">
            <property name="text">
             <string>Remember last used save path</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxSettings">
         <property name="title">
          <string>Torrent settings</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="categoryLayout">
            <item>
             <widget class="QLabel" name="labelCategory">
              <property name="text">
               <string>Category:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="categoryComboBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
              <property name="insertPolicy">
               <enum>QComboBox::InsertAtTop</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item alignment="Qt::AlignRight">
           <widget class="QCheckBox" name="defaultCategoryCheckbox">
            <property name="text">
             <string>Set as default category</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="3" column="0">
             <widget class="QCheckBox" name="doNotDeleteTorrentCheckBox">
              <property name="toolTip">
               <string>When checked, the .torrent file will not be deleted despite the settings at the &quot;Download&quot; page of the options dialog</string>
              </property>
              <property name="text">
               <string>Do not delete .torrent file</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QCheckBox" name="firstLastCheckBox">
              <property name="text">
               <string>Download first and last pieces first</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QCheckBox" name="skipCheckingCheckBox">
              <property name="text">
               <string>Skip hash check</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="sequentialCheckBox">
              <property name="text">
               <string>Download in sequential order</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QCheckBox" name="startTorrentCheckBox">
              <property name="text">
               <string>Start torrent</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="contentLayoutLabel">
              <property name="text">
               <string>Content layout:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="contentLayoutComboBox">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <item>
               <property name="text">
                <string>Original</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Create subfolder</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Don't create subfolder</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="infoGroup">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Torrent information</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0">
           <widget class="QLabel" name="labelDate">
            <property name="text">
             <string>Date:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="labelSizeData"/>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="labelDateData"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="labelSize">
            <property name="text">
             <string>Size:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="labelHashData">
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::TextSelectableByMouse</set>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QScrollArea" name="scrollArea">
            <property name="styleSheet">
             <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>333</width>
               <height>69</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="labelCommentData">
                <property name="textFormat">
                 <enum>Qt::RichText</enum>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
                <property name="openExternalLinks">
                 <bool>true</bool>
                </property>
                <property name="textInteractionFlags">
                 <set>Qt::TextBrowserInteraction</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelHash">
            <property name="text">
             <string>Hash:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelComment">
            <property name="text">
             <string>Comment:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="TorrentContentTreeView" name="contentTreeView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonsHLayout">
     <item>
      <widget class="QCheckBox" name="checkBoxNeverShow">
       <property name="text">
        <string>Never show again</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QHBoxLayout" name="metadataLayout">
       <item>
        <widget class="QProgressBar" name="progMetaLoading">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximum">
          <number>0</number>
         </property>
         <property name="value">
          <number>-1</number>
         </property>
         <property name="textVisible">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblMetaLoading">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonSave">
         <property name="text">
          <string>Save as .torrent file...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionNormal">
   <property name="text">
    <string>Normal</string>
   </property>
  </action>
  <action name="actionHigh">
   <property name="text">
    <string>High</string>
   </property>
  </action>
  <action name="actionMaximum">
   <property name="text">
    <string>Maximum</string>
   </property>
  </action>
  <action name="actionNotDownloaded">
   <property name="text">
    <string>Do not download</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TorrentContentTreeView</class>
   <extends>QTreeView</extends>
   <header>gui/torrentcontenttreeview.h</header>
  </customwidget>
  <customwidget>
   <class>FileSystemPathComboEdit</class>
   <extends>QWidget</extends>
   <header>gui/fspathedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddNewTorrentDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>928</x>
     <y>855</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddNewTorrentDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>928</x>
     <y>855</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboTTM</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>AddNewTorrentDialog</receiver>
   <slot>TMMChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>categoryComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>AddNewTorrentDialog</receiver>
   <slot>categoryChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>savingModeChanged(bool)</slot>
  <slot>categoryChanged(int)</slot>
  <slot>TMMChanged(int)</slot>
 </slots>
</ui>
